charts/alertmanager/values.yaml

```
@@ -84,6 +84,8 @@ affinity: {}
 
 statefulSet:
   annotations: {}
+  podAnnotations: {}
+  command: []
```

```suggestion

podAnnotations: {}

command: []
```

nit: keep these top level